src/Tickets/Commerce/Reports/Data/Order_Summary.php

```diff
@@ -235,6 +251,9 @@ public function get_tickets(): array {
 	 * @return array<string, array{ ticket: Ticket_Object, label: string, type: string, qty_data: array, qty_by_status: string }> A map from ticket types to the Tickets of that type.
 	 */
 	public function get_tickets_by_type(): array {
+		// Sort the tickets by keys to display default types first.
+		ksort( $this->tickets_by_type );
```

Comment:
Meh, hee you're relying on `default` coming before `series_pass` or whatever type; it's not a solid solution as it will have the side-effect of changing if a ticket type is defined that comes before, alpha-numerically speaking, `default`.

Take the robust and verbose route here with an `array_merge`:
```php
$ordered = array_merge(['default' => []], $this->tickets_by_type);
```